temp/legacy-code/src/main/java/com/ivy/legacy/ui/component/transaction/TransactionCard.kt

```diff
@@ -367,6 +366,19 @@ fun CategoryBadgeDisplay(
     }
 }
 
+@Composable
+private fun getTextToShow(transaction: Transaction): String {
```

Comment:
This is a bad naming. What does this method do? Name it accordingly

Reply:
Sure i got so much into checking the stability, renaming it.